apps/mobile/locales/en.json

```diff
@@ -111,6 +113,7 @@
   "Hi {{firstName}}, what would you like to share?": "Hi {{firstName}}, what would you like to share?",
   "Hylo Communities": "Hylo Communities",
   "I agree to the above": "I agree to the above",
+  "If public this post will be visible in the Commons public feed and map": "If public, this post will be visible in the Commons; public feed and map",
```

Comment:
extra ;

Reply:
Intentional. 

"There are treasures in the castle; jewels and gold."